<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="43348" started-at="2022-08-22T15:23:10 IST" finished-at="2022-08-22T15:23:53 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="43348" started-at="2022-08-22T15:23:10 IST" finished-at="2022-08-22T15:23:53 IST">
      <class name="Runner.TestRunner">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:Runner.TestRunner@3fd7a715]" name="beforeTest" is-config="true" duration-ms="7254" started-at="2022-08-22T15:23:10 IST" finished-at="2022-08-22T15:23:17 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="dataExtraction()[pri:1, instance:Runner.TestRunner@3fd7a715]" name="dataExtraction" duration-ms="33062" started-at="2022-08-22T15:23:17 IST" finished-at="2022-08-22T15:23:50 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dataExtraction -->
        <test-method status="PASS" signature="matchCountryOfOrigin()[pri:2, instance:Runner.TestRunner@3fd7a715]" name="matchCountryOfOrigin" duration-ms="16" started-at="2022-08-22T15:23:50 IST" finished-at="2022-08-22T15:23:50 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- matchCountryOfOrigin -->
        <test-method status="FAIL" signature="matchReleaseDate()[pri:2, instance:Runner.TestRunner@3fd7a715]" name="matchReleaseDate" duration-ms="8" started-at="2022-08-22T15:23:50 IST" finished-at="2022-08-22T15:23:50 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Runner.TestRunner.matchReleaseDate(TestRunner.java:65)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- matchReleaseDate -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:Runner.TestRunner@3fd7a715]" name="afterTest" is-config="true" duration-ms="2820" started-at="2022-08-22T15:23:50 IST" finished-at="2022-08-22T15:23:53 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- Runner.TestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
